tailscale/rootfs/etc/s6-overlay/s6-rc.d/post-tailscaled/run

```diff
@@ -5,15 +5,33 @@
 # Runs after the machine has been logged in into the Tailscale network
 # ==============================================================================
 declare -a addresses=()
+declare -a options
 declare -a routes=()
 declare ipinfo
+declare login_server
 declare tags
 
 function appendarray() {
   local -n array=${1}
   readarray -t -O "${#array[@]}" array
 }
 
+# Default options
+options+=(--accept-routes)
+options+=(--advertise-exit-node)
+options+=(--hostname "$(bashio::info.hostname)")
+
+# Get configured control server
+if bashio::config.has_value "login_server";
+then
+  tags=$(bashio::config "login_server")
```

Comment:
@frenck I guess you had a copy paste error here..
Should have been:
`login_server=$(bashio::config "login_server")`

Reply:
🤦  I blame trusting GitHub copilot too much 😓 

Anyways, a PR was opened, I've merged and shipped it 👍 

../Frenck